appsso-starter-java/README.md

```diff
@@ -2,6 +2,35 @@
 
 This repository provides an example application used to set up an authentication mechanism with AppSSO.
 
-## Usage
+## Getting Started
 
+1. Discover your `AppSSO` service offerings
+
+   ```bash
+   tanzu services classes list
+   ```
+
+   If there isn't one,
+
+   - ask your service operator `OR`
+   - [create your own `ClusterUnsafeTestLogin`](https://docs.vmware.com/en/draft/VMware-Tanzu-Application-Platform/1.6/tap/app-sso-reference-api-clusterunsafetestlogin.html)
```

Comment:
That URL gives me a 404, also, we should remove the `/draft/` part here since that is internal only

Reply:
I missed the `draft` piece, thank you. Yes, that does provide a 404, since the docs aren't ready yet. But this is where they will be. I'm not sure what the best way to handle this is right now since the docs don't really exist yet.

Reply:
I will update the `draft` piece. [Here's](https://docs-staging.vmware.com/en/draft/VMware-Tanzu-Application-Platform/1.6/tap/app-sso-reference-api-clusterunsafetestlogin.html) a reference to the staging docs.